From 13bfd439d8b38c16dbaac9fc3fa161cf93d3d0ca Mon Sep 17 00:00:00 2001
From: zzc <zzc@rock-chips.com>
Date: Fri, 18 Aug 2017 17:56:12 +0800
Subject: [PATCH] net: wireless: rockchip_wlan: enable WL_VIRTUAL_APSTA

enable WL_VIRTUAL_APSTA to support concurrent AP and station mode

Change-Id: I217ad4af7989f79aadaf03da08d07d5076ae5834
Signed-off-by: zzc <zzc@rock-chips.com>
---
 drivers/net/wireless/rockchip_wlan/rkwifi/bcmdhd/Makefile | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/net/wireless/rockchip_wlan/rkwifi/bcmdhd/Makefile b/drivers/net/wireless/rockchip_wlan/rkwifi/bcmdhd/Makefile
index a6a4be363d68..e3801d6a4461 100644
--- a/drivers/net/wireless/rockchip_wlan/rkwifi/bcmdhd/Makefile
+++ b/drivers/net/wireless/rockchip_wlan/rkwifi/bcmdhd/Makefile
@@ -99,6 +99,7 @@ DHDCFLAGS += -DWL_SUPPORT_BACKPORTED_KPATCHES
 DHDCFLAGS += -DESCAN_RESULT_PATCH
 DHDCFLAGS += -DVSDB -DWL_CFG80211_VSDB_PRIORITIZE_SCAN_REQUEST
 DHDCFLAGS += -DWLTDLS -DMIRACAST_AMPDU_SIZE=8
+DHDCFLAGS += -DWL_VIRTUAL_APSTA
 endif
 EXTRA_CFLAGS = $(DHDCFLAGS)
 ifeq ($(CONFIG_BCMDHD),m)
-- 
2.35.3

